core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1829,10 +1843,15 @@ class EMConfig {
     @Experimental
     var saveExecutedMainActionInfo = "executedMainActions.txt"
 
-    @Cfg("a probability of enabling single insertion strategy to insert rows into database.")
+
+    @Cfg("Specify prefixes of targets (e.g., MethodReplacement, Success_Call, Local) which will exclude in impact collection. " +
+            "Multiple exclusions should be separated with semicolon (i.e., ;).")
+    @Regex(targetExclusionRegex)
     @Experimental
-    @Probability(activating = true)
-    var probOfEnablingSingleInsertionForTable = 0.0
+    var excludeTargetsForImpactCollection = "Local;MethodReplacement"
```

Comment:
should be based on `ObjectiveNaming` constants

Reply:
this now is updated